src/logger.ts

```diff
@@ -0,0 +1,17 @@
+import { Logger, makeConsoleLogFactory } from "@graphile/logger";
+
+export { LogLevel as MigrateLogLevel } from "@graphile/logger";
+export { Logger as MigrateLogger } from "@graphile/logger";
+export { LogFunctionFactory as MigrateLogFactory } from "@graphile/logger";
```

Comment:
I don't see any need to rename these exports; let's just stick with their normal names :+1: 

```suggestion
export { Logger, LogFunctionFactory, LogLevel } from "@graphile/logger";
```

Reply:
Let's also move this to `index.ts`

```suggestion
```